tests/e2e/upgrade_test.go

```diff
@@ -169,8 +169,7 @@ var _ = Describe("E2E - Upgrading node", Label("upgrade-node"), func() {
 				defer GinkgoRecover()
 
 				By("Checking OS version on "+h+" before upgrade", func() {
-					out, err := client.RunSSH("cat /etc/os-release")
-					Expect(err).To(Not(HaveOccurred()))
+					out := RunSSHWithRetry(cl, "cat /etc/os-release")
```

Comment:
Use `cl` instead of `client`, as it is the variable passed to the function and `client` can be an older/other value instead of the good one!